netket/graph/lattice.py

```diff
@@ -305,6 +307,8 @@ def __init__(
         graph = igraph.Graph()
         graph.add_vertices(len(old_nodes))
         graph.add_edges([(new_nodes[edge[0]], new_nodes[edge[1]]) for edge in edges])
+        graph.simplify()
```

Comment:
does simplify only remove double edges or can it also change the ordering of vertices?

Reply:
I do not think it should reorder anything, similar to `igraph.Graph.delete_edges`. I'd think that would be an `igraph` bug if it does.